archivy/templates/base.html

```diff
@@ -38,190 +34,98 @@ <h3>
+				</a>
+				<a href="/user/edit" class="Header-item">
+					<svg class="octicon" color="white" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="30" height="30"><path fill-rule="evenodd" d="M10.5 5a2.5 2.5 0 11-5 0 2.5 2.5 0 015 0zm.061 3.073a4 4 0 10-5.123 0 6.004 6.004 0 00-3.431 5.142.75.75 0 001.498.07 4.5 4.5 0 018.99 0 .75.75 0 101.498-.07 6.005 6.005 0 00-3.432-5.142z"></path></svg>
 				</a>
-				{{ draw_dir(dataobjs, 0, 0) }}
 			{% endif %}
+			<!-- <div class="toggle Header-item mr-0" style="font-size: 30px">🌓</div> -->
+		</div>
+		<div id="page">
+			{% if current_user.is_authenticated %}
+				<div class="sidebar">
+					{{ draw_dir(dataobjs, 0) }}
+				</div>
+			{% endif %}
+
+			<main class="content pt-3">
+				{% with messages = get_flashed_messages(with_categories=True) %}
+					{% if messages %}
+						{% for category, message in messages %}
+							<div class="flash mt-3 mb-3 flash-{{ category }}">{{ message }}</div>
+						{% endfor %}
+					{% endif %}
+				{% endwith %}
+				{% block content %}{% endblock %}
+			</main>
 		</div>
-        <main class="content">
-            <div><a href="/index">Home</a></div>
-            <h1>Archivy</h1>
-            {% with messages = get_flashed_messages() %}
-                {% if messages %}
-                    <ul>
-                        {% for message in messages %}
-                            <li>{{ message }}</li>
-                        {% endfor %}
-                    </ul>
-                {% endif %}
-            {% endwith %}
-            {% block content %}{% endblock %}
-        </main>
 		
 
-		<script>
-		  /*-----------------------------------------------------*/
-		  /* Different color themes toggle
-		  ------------------------------------------------------ */
-		  const currentTheme = localStorage.getItem("theme");
-		  if (currentTheme == "dark")
-		  {
-			document.body.classList.toggle("dark-theme");
-		  }
+		<!-- 
+			<script>
+			  /*-----------------------------------------------------*/
+			  /* Different color themes toggle
+			  ------------------------------------------------------ */
+			  const currentTheme = localStorage.getItem("theme");
+			  if (currentTheme == "dark")
+			  {
+				document.body.classList.toggle("dark-theme");
+			  }
 
-		  const tog = document.querySelector(".toggle");
-		  tog.addEventListener("click", function () {
-			let theme = "dark";
-			if (document.body.classList.contains("dark-theme"))
-			{
-			  theme = "light";
-			}
-			localStorage.setItem("theme", theme);
-			document.body.classList.toggle("dark-theme");
-		  });
-			
-		</script>
+			  const tog = document.querySelector(".toggle");
+			  tog.addEventListener("click", function () {
+				let theme = "dark";
+				if (document.body.classList.contains("dark-theme"))
+				{
+				  theme = "light";
+				}
+				localStorage.setItem("theme", theme);
+				document.body.classList.toggle("dark-theme");
+			  });
+				
+			</script>
+		-->
```

Comment:
remove commented code, if its no longer useful. No explanation of why it has been commented is given, perhaps we should just remove it